Tests/Tests/CDTURLSessionTests.m

```diff
@@ -0,0 +1,62 @@
+//
+//  CDTURLSessionTests.m
+//  Tests
+//
+//  Created by Rhys Short on 24/08/2015.
+//
+//
+
+#import <XCTest/XCTest.h>
+#import "CloudantSyncTests.h"
+#import "CDTURLSession.h"
+#import "CDTHTTPInterceptorContext.h"
+#import "CDTHTTPInterceptor.h"
+
+@interface CDTNoOpRequestInterceptor : NSObject<CDTHTTPInterceptor>
+
+@property int timesCalled;
+
+@end
+@implementation CDTNoOpRequestInterceptor
+
+
+
+- (nonnull CDTHTTPInterceptorContext *)interceptRequestInContext:(nonnull CDTHTTPInterceptorContext *)context {
+    self.timesCalled++;
+    return context;
+}
+
+@end
+
+@interface CDTURLSessionTests : CloudantSyncTests
+
+@end
+
+
+@implementation CDTURLSessionTests
+
+-(void) testRequestInterceptorsSetCorrectly {
+    
+    CDTNoOpRequestInterceptor * noOp = [[CDTNoOpRequestInterceptor alloc] init];
```

Comment:
`CDTNoOpHttpInterceptor *noOp` -- spacing and rename the test object in light of changing the protocol name.